pkg/label/label.go

```diff
@@ -102,6 +102,15 @@ const (
 	// AnnTiKVLastAutoScalingTimestamp is annotation key of tidbclusterto which ordinal is created by tikv auto-scaling
 	AnnTiKVLastAutoScalingTimestamp = "tikv.tidb.pingcap.com/last-autoscaling-timestamp"
 
+	// AnnTiDBConsecutiveScaleOutCount describes the least consecutive count to scale-out for tidb
```

Comment:
least or largest?

Reply:
least, the recorded consecutive count should be bigger than `AnnTiDBConsecutiveScaleOutCount` to perform auto-scaling out